contracts/rewards/VestingRecipient.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import { AuraMath } from "../utils/AuraMath.sol";
+import { IAuraLocker } from "../interfaces/IAuraLocker.sol";
+import { IERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/utils/SafeERC20.sol";
+import { Initializable } from "@openzeppelin/contracts-0.8/proxy/utils/Initializable.sol";
+
+interface IVestedEscrow {
+    function claim(bool) external;
+
+    function rewardToken() external view returns (address);
+
+    function totalClaimed(address) external view returns (uint256);
+}
+
+/**
+ * @title   VestingRecipient
+ * @author  AuraFinance
+ */
+contract VestingRecipient is Initializable {
+    using AuraMath for uint256;
+    using SafeERC20 for IERC20;
+
+    // ----------------------------------------------------------
+    // Storage
+    // ----------------------------------------------------------
+
+    uint256 public unlockTime;
```

Comment:
The is no setter for `unlockTime` ,  so it could be change to immutable , right ?